pkgs/development/tools/rust/rust-analyzer/default.nix

```diff
@@ -1,30 +1,41 @@
-{ lib
-, stdenv
-, callPackage
-, fetchFromGitHub
-, rustPlatform
-, CoreServices
-, cmake
-, libiconv
-, useMimalloc ? false
-, doCheck ? true
-, nix-update-script
+{
+  lib,
+  stdenv,
+  callPackage,
+  fetchFromGitHub,
+  rustPlatform,
+  CoreServices,
```

Comment:
I believe this can be replaced by the new `darwin.apple_sdk_*` since #346043 (TL;DR: #354146). But I don't have a darwin machine. You can leave it as is if you cannot test it.

If this can be removed, we can also migrate `rust-analyzer*` to `pkgs/by-name`.

Reply:
I can do that, I didn't at first because I wasn't sure if you'd be concerned about the signature change during the end of the release cycle.